syspage.h

```diff
@@ -77,7 +77,7 @@ extern void syspage_progShow(void);
 extern void syspage_consoleSet(unsigned int id);
 
 
-#if HAS_GRAPHICS
+#ifdef HAS_GRAPHICS
```

Comment:
`HAS_GRAPHICS` is used as a 0/1 toggle, like the one in `hal/ia32/config.c`:
```c
/* Platform can set graphic modes (i.e. via VBE) */
#define HAS_GRAPHICS 1
```

Maybe instead of changing `if` to `ifdef` we could guarantee a symbol is always defined in `syspage.h` with a sane default?
```c
#ifndef HAS_GRAPHICS
#define HAS_GRAPHICS 0
#endif
```